src/util/grb_net.cc

```diff
@@ -75,7 +75,7 @@ GrbNet::GrbNet(const std::string& addr, int af)
 
 GrbNet::GrbNet(const struct sockaddr_storage* addr)
 {
-    std::memcpy(&sockAddr, addr, sizeof(sockAddr));
+    sockAddr = *addr;
```

Comment:
You have to use `copy_n` as well, otherwise you may copy the `ss_data` pointer intead of the array contents:

```
 struct sockaddr_storage {
     sa_family_t ss_family;
     uint8_t ss_data[SOCKADDR_MAX_DATA_LEN]; 
 };
```

Reply:
Looks the same to me:

https://godbolt.org/z/84Man3GMW